add bytesize SIG regulatory (#2734)
nf-core/website: sites/main-site/src/content/events/2025/bytesize_sig_regulatory.md

---
title: 'Bytesize: Special Interest Group - regulatory'
subtitle: Alex Peltzer, Boehringer Ingelheim
type: talk
startDate: '2025-01-28'
startTime: '13:00+02:00'
endDate: '2025-01-28'
endTime: '13:30+02:00'
locations:
  - name: Online
    links:
      - https://kth-se.zoom.us/j/68390542812
---

> In this nf-core bytesize session, Alex ([@apeltzer](https://github.com/apeltzer)) will talk all about the regulatory special interest group and provide some updates on what this is about, what it tries to tackle & solve and share some updates on what has been worked upon already. This group is a place to discuss the subject around regulatory & validation topics for analysis pipelines and how nf-core and the community as such can facilitate validation of nf-core pipelines as such by providing improved resources and documentation to perform such tasks but also considering adding tooling to help with this in the future. Its still early days but the bytesize is an excellent place to catch up and learn about this first hand!

Reviewer: ```suggestion
In this nf-core bytesize session, Alex ([@apeltzer](https://github.com/apeltzer)) will talk about the regulatory special interest group and provide some updates on what this group is about, what the group tries to tackle and share some updates on what has been worked upon already. This group is a place to discuss the subject regulatory & validation for analysis pipelines and how nf-core and the community as such can facilitate any efforts in this direction by providing improved resources and documentation to perform such tasks but also considering adding tooling to help with this in the future. Its still early days but the bytesize is an excellent place to catch up and learn about this first hand!
```
